### ISSUE TITLE
Launching Oculus Library Games with Arguments

### ISSUE THREAD

**Issue by IzNotBen** [LibreVR/Revive#1827]
**Which game or application is your request about?**
- "Any Oculus Game"

**Is your request related to a problem? Please describe.**
- No, I am wondering if there is a way to launch Oculus Library games with arguments.

**Describe the solution you'd like**
- Some games allow you to change how they start up my using command arguments in a shortcut, .bat file, or in the command line in the specific directory. I am curious if there is any way to support this kind of feature or if it already exists.

**Describe alternatives you've considered**
- There are no alternatives.

Happy Trails!

**Comment by miketunes** [comment]
Yes, I'm trying to launch after the fall with a bhapics mod, but it only launches in flat screen:
https://www.nexusmods.com/untilyoufall/mods/1?tab=description
